### PR TITLE
feat: landing page added

### PR DESCRIPTION
**Pull request by mehulsatardekar**

### CODE REVIEW

File: src/pages/quiz-landing/Landing.tsx [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          </h3>
+          <p className="home-txt-descr">
+            We help you prepare for exams and quizzes
+          </p>
+          <Link to="/" className="btn-primary btn btn-py-1">
+            Start Learning
+          </Link>
+        </div>
+      </section>
+      <section className="home-banner"></section>
```
Comment (gauravsinhaweb) [inline src/pages/quiz-landing/Landing.tsx:22]: if it is not in use then remove it

File: src/pages/quiz-landing/landing.css [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+}
+
+.home-txt-descr {
+  font-size: 1rem;
+  border-left: 2px solid #46178f;
+  padding-left: 3px;
+}
+.home-banner {
+  flex: 1;
+  background: #46178f;
```
Comment (gauravsinhaweb) [inline src/pages/quiz-landing/landing.css]: variable color!

File: src/pages/quiz-landing/landing.css [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+.home-txt {
+  font-size: 1.5rem;
+  font-weight: 800;
+  color: var(--ex-dark-blue);
+}
+
+.home-txt-descr {
+  font-size: 1rem;
+  border-left: 2px solid #46178f;
```
Comment (gauravsinhaweb) [inline src/pages/quiz-landing/landing.css]: use var color

**Review by gauravsinhaweb — APPROVED** [review]

**Review by Ankur9669 — APPROVED** [review]
Code looks good to me

**Review by MarkVed17 — APPROVED** [review]
Code formatting could be done better. Extra line spaces at a few places in your files. Remove those and you’re good to proceed.

File: src/pages/quiz-landing/landing.css [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  .home-banner {
+    display: none;
+  }
+
+  .home-txt {
+    font-size: 1.3em;
+  }
+
+  .home-txt-container {
+    background: url("https://images.weserv.nl/?url=https://res.cloudinary.com/dwhsfh3sc/image/upload/v1654103604/quiz-grad/assets/Illusttration_o3d6wd.png")
```
Comment (bajajcodes) [inline src/pages/quiz-landing/landing.css:59]: use some file for storing static data.

**Review by bajajcodes — APPROVED** [review]

**Pull request merged**